crates/nu-cmd-extra/tests/commands/bits/into.rs

```diff
@@ -0,0 +1,13 @@
+use nu_test_support::nu;
+
+#[test]
+fn byte_stream_into_bits() {
+    let result = nu!("[0x[01] 0x[02 03]] | bytes collect | into bits");
```

Comment:
Shouldn't these just work without the `bytes collect`? Seems similar to the problem with `into binary` where it didn't work without collect until you fixed it yesterday.

Reply:
In this case I'm just specifically testing the behavior on a stream of bytes, so `bytes collect` is there to make that specific type of input.

Reply:
ah, ok. thx